lib/tasks/send_missing_addresses_csv.rake

```diff
@@ -0,0 +1,78 @@
+namespace :correct_addresses do
+  # rubocop:disable Lint/ConstantDefinitionInBlock
+  MISSING_ADDRESSES_THRESHOLD = 50
+  # rubocop:enable Lint/ConstantDefinitionInBlock
+
+  desc "Send missing lettings addresses csv"
+  task :send_missing_addresses_lettings_csv, %i[] => :environment do |_task, _args|
+    Organisation.all.each do |organisation|
+      logs_impacted_by_missing_address = organisation.managed_lettings_logs
```

Comment:
I feel like we're using similar combinations of scopes a lot that could be combined into two or three reusable compound scopes

Reply:
I've combined a couple, not sure if extracting the rest would make it clearer, especially as this is quite specific to this task. Let me know what you think!

Reply:
Yeah the ones you've done are all the ones I think are worth doing, love it 🚀